webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -762,20 +751,33 @@ class EligibilityMaybeDisplaySteuerlotseStep(EligibilityStepMixin, DisplaySteuer
     intro = _l('form.eligibility.success.maybe.intro')
     template = 'eligibility/display_maybe.html'
 
-    def __init__(self, endpoint, stored_data=None, *args, **kwargs):                
+    def __init__(self, endpoint, stored_data=None, *args, **kwargs):               
         super(EligibilityMaybeDisplaySteuerlotseStep, self).__init__(endpoint=endpoint,
-                                                                     stored_data=stored_data,
-                                                                     header_title=_('form.eligibility.header-title'),
-                                                                     *args, **kwargs)        
+                                   stored_data=stored_data,
+                                   header_title=_('form.eligibility.header-title'),
+                                   *args,
+                                   **kwargs)
         
         
     def _main_handle(self):  
         super()._main_handle()
-        dependent_notes = [(_('form.eligibility.result-note.capital_investment'))]
-        dependent_notes.append(_('form.eligibility.result-note.deadline'))
               
-        self.render_info.additional_info['dependent_notes'] = dependent_notes
+        self.render_info.additional_info['dependent_notes'] = get_dependent_note(self)
         self.render_info.detail = {'render_kw': {
                 'data-detail': {'title': _l('form.eligibility.result-note.when_unlock_code.title'),
                                 'text': _l('form.eligibility.result-note.when_unlock_code.description')}}
                 }  
+
+
+def get_dependent_note(self):                 
```

Comment:
Can we name this get_dependent_notes as it returns a list?

Reply:
Done!